demos/multigrid/geometric_multigrid.py.rst

```diff
@@ -167,9 +167,16 @@ remembering to tell PETSc to use pivoting in the factorisation. ::
                                                "ksp_monitor": True,
                                                "pmat_type": "aij"})
 
-Next we'll use a schur complement solver, using geometric multigrid to
+Next we'll use a Schur complement solver, using geometric multigrid to
 invert the velocity block. ::
 
+  class Mass(ExplicitSchurPC):
+
+      def form(self, test, trial, pc):
+          a = inner(test, trial)*dx
+          bcs = None
+          return (a, bcs)
+
```

Comment:
You could now address the note lower down, and push the mass matrix into the coarse grid fieldsplit in the monolithic case.

Reply:
Done. I also used the mass matrix in the smoothers on the finer levels, not sure if that's what you wanted.